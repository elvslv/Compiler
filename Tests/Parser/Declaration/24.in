procedure proc(var i: integer); forward;
procedure proc0(var i, m, k: integer; s: float); forward;
procedure proc1(var j: float): integer; forward;