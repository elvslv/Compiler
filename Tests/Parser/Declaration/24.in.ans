procedure PROC(var (by ref) I : INTEGER;);
procedure PROC0(var (by ref) I : INTEGER;var (by ref) K : INTEGER;var (by ref) M : INTEGER;var S : FLOAT;);
30 3 ; expected