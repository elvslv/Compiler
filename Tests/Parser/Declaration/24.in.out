procedure PROC(var (by ref) I : INTEGER;);procedure PROC0(var (by ref) I : INTEGER;var (by ref) K : INTEGER;var (by ref) M : INTEGER;var S : REAL;);
29 3 ; expected end of file, but : found